src/conn/conn_cache.c

```diff
@@ -46,6 +46,111 @@ __cache_config_abs_to_pct(
     return (0);
 }
 
+/*
+ * __validate_cache_config --
+ *     Validate trigger and target values of given configs.
+ */
+static int
+__validate_cache_config(WT_SESSION_IMPL *session, const char *cfg[], bool create)
+{
+    WT_CACHE *cache;
+    WT_CONFIG_ITEM cval;
+    WT_CONNECTION_IMPL *conn;
+    bool debug_config;
+
+    conn = S2C(session);
+    cache = conn->cache;
+
+    WT_RET(__wt_config_gets(session, cfg, "debug_mode.configuration", &cval));
+    debug_config = cval.val;
+
+    /* The target size must be lower than the trigger size or we will never get any work done. */
+    if (create) {
+        if (cache->eviction_target >= cache->eviction_trigger)
+            WT_RET_MSG(session, EINVAL, "eviction target must be lower than the eviction trigger");
+        if (cache->eviction_dirty_target >= cache->eviction_dirty_trigger)
+            WT_RET_MSG(session, EINVAL,
+              "eviction dirty target must be lower than the eviction dirty trigger");
+        if (cache->eviction_updates_target >= cache->eviction_updates_trigger)
+            WT_RET_MSG(session, EINVAL,
+              "eviction updates target must be lower than the eviction updates trigger");
+        return (0);
+    }
+
+    WT_RET(__wt_config_gets(session, cfg, "eviction_target", &cval));
+    cache->eviction_target = (double)cval.val;
```

Comment:
It is confusing to see that `cache->eviction_target` was just checked above and now we're setting it. Maybe add a comment after the conditional that now we're resetting the values via reconfigure or restart or however we get here.

Reply:
sounds good, comment added in: [c7c0742](https://github.com/wiredtiger/wiredtiger/pull/10548/commits/c7c074299c46b9cb2a5ceb857089d5053db06b33)

Reply:
I've removed the `create=true` logic and the second call to` __validate_cache_config(create=true)` in: [9e4baa7](https://github.com/wiredtiger/wiredtiger/pull/10548/commits/9e4baa7696ed61b4507e4766121155606277fcd8)